RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate PickerView to Typescript

## Proposed changes

Migrate PickerView to Typescript

## Issue(s)	


## How to test or reproduce

- You can access the PickerView going to Notification Preferences inside the actions of room, then click on any button below the section `Show Unread Counter`. So when you click on any of them, you should be moved to another screen with a list of options, and clicking on any of the options, you should go back to previous screen
-  or going to Profile, then click on Preferences, then `Notifications` and any of the buttons should have the same behavior as above

## Screenshots

## Types of changes




## Checklist



## Further comments
